pkg/collector/process.go

```diff
@@ -42,8 +43,10 @@ type collectingProcess struct {
 	maxBufferSize uint16
 	// chanel to receive stop information
 	stopChan chan bool
-	// packet list
-	messages []*entities.Message
+	// useMsgChan indicates whether to use message channel to output message
+	useMsgChan bool
```

Comment:
does useMsgChan have any other use case than usage in unit tests?

Reply:
I cannot figure out any other. I suppose we can remove it.